lib/emque/producing/message/message.rb

```diff
@@ -130,16 +128,20 @@ def to_json
         Oj.dump(data, :mode => :compat)
       end
 
-      def publish(publisher=nil)
-        publisher ||= Emque::Producing.publisher
+      def publish(publishers=[:rabbitmq])
```

Comment:
seems odd to be setting a default publisher in the gem. Can we either extract this to a config in init or just remove it and let it blow up if someone tries to call publish without a default?

Reply:
conceptually:
```
  c.default_publisher = :rabbitmq
```

Reply:
or just throw away defaults.... keep it simple.